Source/Resources/Content/AzureManagement/MicrosoftIntune/DiagnosticSettings/Logging.json

```diff
@@ -1,4 +1,5 @@
 {
+  "description":"What is it: Configures logging to capture Intune events, which can then be analyzed using Log Analytics Workspaces. \n Simeon Notes: Provides you with the data needed to monitor Intune events within your tenant. Setup alerts when a device is marked as non-compliant or be notified when a new device is enrolled. Simeon's baseline captures device audit logs, compliance, and operational logs. Logs can be viewed in the Analytics Workspace named 'tenant Name-M365logging' and are stored in the Azure Storage Account named tenantNameM365Logging.\n Learn More: https://techcommunity.microsoft.com/t5/device-management-in-microsoft/microsoft-intune-and-azure-log-analytics/ba-p/463145",
```

Comment:
**What does this do?**
Configures logging to capture Intune events, which can then be analyzed using Log Analytics Workspaces. The baseline captures device audit logs, compliance, and operational logs. Logs can be viewed in the Analytics Workspace named 'tenant Name-M365logging' and are stored in the Azure Storage Account named tenantNameM365Logging.

Setup alerts when a device is marked as non-compliant or be notified when a new device is enrolled. -- @metamenos where should this bit be added? Not sure what it means 

**Why should you use this?**
Improves security by providing you with the data needed to monitor Intune events within your tenant. 

**What is the end-user impact?**
N/A

**Learn more**
[Microsoft Intune and Azure Log Analytics](https://techcommunity.microsoft.com/t5/device-management-in-microsoft/microsoft-intune-and-azure-log-analytics/ba-p/463145)

Reply:
@david-simeon  added the part in question to Why should you use this.